Integrations/PaloAltoNetworksCortex/PaloAltoNetworksCortex_test.py

```diff
@@ -80,3 +97,10 @@ def test_prepare_fetch_query(mocker):
     assert firewall_query_with_severity_and_subtype == "SELECT * FROM panw.threat WHERE receive_time>1524383011 " \
                                                        "AND (subtype='url' OR subtype='antivirus') " \
                                                        "AND (severity='medium')"
+
+
+def test_get_encrypted():
+    from PaloAltoNetworksCortex import get_encrypted
+    auth_id = random_string(50)
+    auth_key = random_string(32)
+    get_encrypted(auth_id, auth_key)
```

Comment:
need to check that the actual value is the expected, so lets not do it with random string

Reply:
the nonce (random int) is always random (we should never use the same nonce twice) so we don't know what will go out

Reply:
I can add a decryption function to the test.